### PR TITLE
fix(opinions.css):Front end tweaks

### PR DESCRIPTION
**Pull request by flooie**
Remove pink/brown caption color
Go Sans Serif

@mlissner a nice gray - and sans serif for your approval 

Screenshot 2024-12-12 at 3 10 46 PM

### CODE REVIEW

File: cl/assets/static-global/css/opinions.css [hunk 1]
```diff
@@ -6,7 +6,7 @@
 .opinion-body {
 
   .harvard > * {
-    font-family: Merriweather, "Times New Roman", Times, serif;
+    font-family: "Source Sans Pro", Arial, Helvetica, sans-serif;
```
Comment (mlissner) [inline cl/assets/static-global/css/opinions.css:9]: Do you need this? Can't you just use the default?

**Review by mlissner — APPROVED** [review]
One thought for you.

File: cl/assets/static-global/css/opinions.css (hunk 1 above)
Comment (flooie) [reply cl/assets/static-global/css/opinions.css:9]: In theory no. but I personally find the default font to be off putting .

Comment (flooie) [reply cl/assets/static-global/css/opinions.css:9]: Of course I am happy to do what you suggest.

Comment (mlissner) [reply cl/assets/static-global/css/opinions.css:9]: Not the time to discuss or think about fonts. Please remove.

**Comment by flooie** [comment]
@mlissner why did you assign this back to me?

**Review by mlissner — APPROVED** [review]

**Comment by mlissner** [comment]
Into the heap it goes! Thank you!

**Comment by flooie** [comment]
thank you

**Pull request merged**